src/sample_submission_data.R

```diff
@@ -0,0 +1,88 @@
+## Script that generates sample submission data to pass validation checks for
+## the forecasting hub
+## It's not particularly efficient but it's easy to understand
+
+## Function to generate simulated data
+# Uses the Dirichlet distribution to generate random proportions of clades
+require(brms) # for data simulation
+require(arrow) # for parquet writing
+
+
+sim_model_output <- function(locations = c("AL", "AZ", "CA", "CO", "DE"), 
+                             alpha = c(3,3,2,2,1,1),
+                             test_clades = c("24A", "24B", "24C", "24E", "recombinant", "other"),
+                             target_date = seq(as.Date("2024-09-01"), as.Date("2024-10-12"), by = "1 days"),
+                             nowcast_date = "2024-10-02",
+                             n_samp = 100,
+                             seed = 42
+){
+  ## Simulate data in appropriate model_output format
```

Comment:
Minor comment, but I think the Roxygen format requires this documentation chunk to be before the function definition.